### PR TITLE
Footer update

### LINKED ISSUE #94: Footer
Create a gorgeous footer. Don't forget to mention our community!

### PR DESCRIPTION
**Pull request by DannySS123**
Not great, not terrible
image
Waiting for better ideas
Closes #94

### CODE REVIEW

**Review by Tschonti — APPROVED** [review]

**Comment by berenteb** [comment]
There are unnecessary elements, I have some ideas for this:
- Only display the link for the Impressum page, "ha érdeklődsz..." not needed
- Facebook and Instagram icons are enough, no label needed. The user will now what they do.
- "Esemény szervezője" label not needed IMO.
- Maybe a mail icon instead of the whole e-mail address, or only the contact link.
- Smaller logos

This three section layout is okay though.
It is indeed a tough job to get it right!
It is big. "ÚRISTEN..."

**Comment by berenteb** [comment]
Képernyőfotó 2022-01-26 - 13 38 08
How about this?

File: src/main/client/src/utils/configurations.ts [hunk 1]
```diff
@@ -5,3 +5,5 @@ export const BACKEND_BASE_URL = (() => {
 export const API_BASE_URL = process.env.NODE_ENV === 'development' ? 'localhost:8080' : ''
 export const KIRDEV_URL = process.env.REACT_APP_KIRDEV_URL || 'https://kir-dev.sch.bme.hu'
 export const BUGREPORT_URL = process.env.REACT_APP_BUGREPORT_URL || 'https://kir-dev.sch.bme.hu/about#contact'
+
+export const KIRDEV_ORANGE = '#F15A29'
```
Comment (berenteb) [inline src/main/client/src/utils/configurations.ts:9]: Putting this inside theme?

File: src/main/client/src/utils/customTheme.ts [hunk 2]
```diff
@@ -1,8 +1,14 @@
...
+import { mode, createBreakpoints } from '@chakra-ui/theme-tools'
 
 // See more: https://chakra-ui.com/docs/theming/customize-theme
 const customTheme = extendTheme({
+  breakpoints: createBreakpoints({
+    sm: '30em',
+    md: '48em',
+    lg: '68em',
+    xl: '80em'
+  }),
```
Comment (berenteb) [inline src/main/client/src/utils/customTheme.ts:11]: What happened to the old one? Seems fine, just curious.

File: src/main/client/src/components/@layout/Footer.tsx [hunk 3]
```diff
@@ -1,13 +1,86 @@
...
+                    <HStack pb={2} as={Link} _hover={{ color: KIRDEV_ORANGE }} href={sociaPage.href} isExternal key={sociaPage.label}>
+                      <Text>{sociaPage.label}</Text>
+                      <Icon as={sociaPage.icon} boxSize="1.5rem" />
+                    </HStack>
+                  ))}
+                </Flex>
+              </Flex>
+            </Flex>
+          </Center>
+        </WrapItem>
```
Comment (berenteb) [inline src/main/client/src/components/@layout/Footer.tsx:59]: There are unuseful layers of containers. Same result can be achieved by only one VStack and 1-2 Flex components. Also I wouldn't recommend fixing the size of one column.

**Review by berenteb — COMMENTED** [review]
I have fixed most of these problems with the new concept I just commented. I commit them, see if those are usable.

File: src/main/client/src/utils/customTheme.ts (hunk 2 above)
Comment (Tschonti) [reply src/main/client/src/utils/customTheme.ts:11]: By default, the lg breakpoint was at 62em, but we wanted the middle column to be the first to wrap if the screen was smaller, and the first wrapping occurred at 68em, but this value depends on the width of the columns

Comment (DannySS123) [reply src/main/client/src/utils/customTheme.ts:11]: Only the lg was modified, but with your changes I think it should work without this.

Comment (berenteb) [reply src/main/client/src/utils/customTheme.ts:11]: Yesssss it does. The 100% width of columns below md size will force the layout to wrap.

File: src/main/client/src/components/@layout/Footer.tsx [hunk 4]
```diff
@@ -1,13 +1,83 @@
...
+}
+const ImpressumWrapItem: React.FC<impresszumWrapItemProps> = ({ display, key }) => {
+  return (
+    <FooterWrapItem key={key} display={display}>
+      <Text marginBottom={3} align="center" fontSize="xl">
+        <Link textColor={KIRDEV_ORANGE} href="/impresszum">
+          Impresszum
+        </Link>
+        {' | '}
+        <Link textColor={KIRDEV_ORANGE} isExternal href="https://www.youtube.com/watch?v=YLR6tqJDHEc">
```
Comment (poiuztrewq1) [inline src/main/client/src/components/@layout/Footer.tsx:23]: Annyira Rickroll-ra számítottam...
1pcs5p

**Review by poiuztrewq1 — APPROVED** [review]
Overall nice work. I really like this big footer with the big Kir-Dev logo.

File: src/main/client/src/components/@layout/Footer.tsx (hunk 4 above)
Comment (berenteb) [reply src/main/client/src/components/@layout/Footer.tsx:23]: Get ready for Jacksonrole

**Pull request merged**